README.md

```diff
@@ -36,28 +38,21 @@ This Library is in preview and hasn't been released yet. Some features may not w
 
 Get the library from the Python Package Index (PyPI)
 <br>
+
 ```
 pip install EpikCord.py
 ```
 
-### Install in an Virtual Environment (venv)
```

Comment:
Keep this in here...

Reply:
Wouldnt it be fair if we do it in a separate file (docs/installing.rst)

Reply:
Won't be visible for a long time

Reply:
wdyjm?